src/classes/BDI_DataImportService.cls

```diff
@@ -78,18 +88,21 @@ global with sharing class BDI_DataImportService {
     * @return String the soql String
     */
     public static String strSoqlForDataImportProcess(List<Id> dataImportIds) {
+        List<String> whereClauses = new List<String>{
+            'Status__c <> \' + BDI_DataImport_API.bdiImported + \'',
+            'Id =: dataImportIds'
+        };
         String dataImportQuery;
```

Comment:
Could we go ahead and just scrub `dataImportQuery` and immediately return `new UTIL_Query()...`?

Reply:
agreed! done!